recipes/glib/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# don't get locally installed pkg-config entries:
+#export PKG_CONFIG_LIBDIR="${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig"
+
+# Needed to get appropriate response to g_get_system_data_dirs():
+export CFLAGS="-DCONDA_SYSTEM_DATA_DIRS=\\\"${PREFIX}/share\\\""
+
+if [ "$(uname)" == "Darwin" ] ; then
+  # for Mac OSX
+  export CC=clang
+  export CXX=clang++
+  # Cf. the discussion in meta.yaml -- we require 10.7.
+  export MACOSX_DEPLOYMENT_TARGET="10.7"
+  SDK=/
+  export CFLAGS="${CFLAGS} -isysroot ${SDK}"
+  export LDFLAGS="${LDFLAGS} -Wl,-syslibroot,${SDK}"
+  # Pick up the Conda version of gettext/libintl:
+  export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+  export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib -Wl,-rpath,${PREFIX}/lib"
+else
+  # for linux
+  # Pick up the Conda version of gettext/libintl:
+  export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+  export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+fi
+
+./configure --prefix=${PREFIX} --with-python="${PYTHON}" --with-libiconv=gnu \
+  || { cat config.log; exit 1; }
+make
```

Comment:
There is an extraneous error that causes the check to fail. Can't find any info about it, so it is probably a bug

```
PASS: fileutils 1 /fileutils/build-path
PASS: fileutils 2 /fileutils/build-pathv
PASS: fileutils 3 /fileutils/build-filename
PASS: fileutils 4 /fileutils/build-filenamev
PASS: fileutils 5 /fileutils/mkdir-with-parents
PASS: fileutils 6 /fileutils/format-size-for-display
PASS: fileutils 7 /fileutils/errors
PASS: fileutils 8 /fileutils/basename
PASS: fileutils 9 /fileutils/dir-make-tmp
PASS: fileutils 10 /fileutils/file-open-tmp
PASS: fileutils 11 /fileutils/mkstemp
PASS: fileutils 12 /fileutils/mkdtemp
PASS: fileutils 13 /fileutils/set-contents
PASS: fileutils 14 /fileutils/read-link
ERROR: fileutils - too few tests run (expected 15, got 14)
ERROR: fileutils - exited with status 134 (terminated by signal 6?)
```